### PR TITLE
FE 39 and FE 41 - Implement PCV balance in dashboard stats panel

### LINKED ISSUE #39: Change the button text of the Open Vault card in the Dashboard
The button text should change to "Adjust Vault" or "Liquidated Vault" depending on the specific stage of the user's current vault. 
image

### LINKED ISSUE #41: Implement PCV balance in the dashboard stats card
Implement PCV balance in the dashboard stats card

### PR DESCRIPTION
**Pull request by evandrosaturnino**
[feat: implement pcv balance in stats panel](https://github.com/Threshold-USD/dev/commit/2305624fc9868dea25b83934f7952157665240d9) - closes #41 

[feat: dashboard vault caption change based on view](https://github.com/Threshold-USD/dev/commit/09c01d2f0598304f98740a5480e8e7ad7190ff6d) - closes #39

### CODE REVIEW

File: packages/dev-frontend/src/LiquityFrontend.tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ const UnsupportedMainnetFallback: React.FC = () => (
     </Heading>
 
     <Paragraph sx={{ mb: 3 }}>
-      Please change your network to Ropsten.
+      Please change your network to Rinkeby.
```
Comment (benlongstaff) [inline packages/dev-frontend/src/LiquityFrontend.tsx]: Rinkeby is going to shut down in Q4 2022 we should probably switch to Goerli

File: packages/contracts/utils/priceFeedInteractions.js [hunk 2]
```diff
@@ -135,6 +135,6 @@
 //     Therefore, expected gas cost of a getLatestPrice() call is within a Trove function is (35k - 27.5k) 
 //     = 7500 gas upper bound.
     
-//     To check, deploy an instance of FunctionCaller contract to ropsten and mainnet, 
+//     To check, deploy an instance of FunctionCaller contract to rinkeby and mainnet,
```
Comment (benlongstaff) [inline packages/contracts/utils/priceFeedInteractions.js:138]: Rinkeby is going to shut down in Q4 2022 we should probably switch to Goerli

File: packages/dev-frontend/src/components/SystemStatsCard.tsx [hunk 3]
```diff
@@ -116,17 +119,25 @@ export const SystemStatsCard: React.FC<SystemStatsCardProps> = ({ variant = "inf
...
+            tooltip={`The total ${ COIN } currently held in the PCV, expressed as an amount and a fraction of the ${ COIN } supply.`}
           >
-            {total.debt.shorten()}
+            {pcvBalance.prettify()}
           </SystemStat>
           <SystemStat 
-            info="Recovery Mode" 
-            tooltip="Recovery Mode is activated when the Total Collateral Ratio (TCR) falls below 150%. When active, your Vault can be liquidated if its collateral ratio is below the TCR. The maximum collateral you can lose from liquidation is capped at 110% of your Trove's debt. Operations are also restricted that would negatively impact the TCR." 
+            info={`${ COIN } Supply`} 
+            tooltip={`The total ${ COIN } minted by the ThresholdUSD Protocol.`}
```
Comment (benlongstaff) [inline packages/dev-frontend/src/components/SystemStatsCard.tsx]: missing a space in Threshold USD

**Review by benlongstaff — APPROVED** [review]

File: packages/dev-frontend/src/components/SystemStatsCard.tsx (hunk 3 above)
Comment (evandrosaturnino) [reply packages/dev-frontend/src/components/SystemStatsCard.tsx]: Resolved

File: packages/contracts/utils/priceFeedInteractions.js (hunk 2 above)
Comment (evandrosaturnino) [reply packages/contracts/utils/priceFeedInteractions.js:138]: Resolved

File: packages/dev-frontend/src/LiquityFrontend.tsx (hunk 1 above)
Comment (evandrosaturnino) [reply packages/dev-frontend/src/LiquityFrontend.tsx]: Resolved

**Comment by benlongstaff** [comment]
LGTM

**Pull request merged**